pkg/metrics/otel.go

```diff
@@ -51,6 +51,19 @@ exporters:
       # The metric streaming data is not affected
       # https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/issues/529
       skip_create_descriptor: true
+      resource_filters:
```

Comment:
Add a comment explaining why we're filtering these.

Reply:
Done